05week/spaceTravelToMars.js

```diff
@@ -10,6 +19,65 @@ let jobTypes = {
 };
 
 // Your code here
+class Ship {
+  constructor(name, type, ability, crew) {
+    this.name = name;
+    this.type = type;
+    this.ability = ability;
+    this.crew = [];
+
+  }
+  missionStatement() {
+    return this.ability;
```

Comment:
this is almost correct, however it doesn't satisfy the tests, the ability is only returned if there is a crew member on the ship